stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -2048,7 +2048,7 @@ public struct Dictionary<Key : Hashable, Value> :
     }
     mutableAddressWithNativeOwner {
       let (_, address) = _variantBuffer
-        .pointerToValue(forKey: key, insertingDefault: defaultValue)
+        .pointerToValue(forKey: key, insertingDefault: defaultValue())
```

Comment:
Could we remove the `@autoclosure` attribute from the `_VariantDictionaryBuffer` methods instead? Without being able to see the signature for `pointerToValue` it really looks like `defaultValue` is getting called here.

Reply:
I didn't have any strong opinion on it at the time; but now you mention it, yup, I agree.